// Another service node that subscribes to the robot position and velocity(using the custom message)and implements a server to retrieve the distance of the robot from the target and the robot average speed.

float distance
float average_speed
